webapp/src/components/Navbar/Navbar.js

```diff
@@ -49,6 +65,20 @@ export default class Navbar extends React.PureComponent {
     event.preventDefault()
   }
 
+  handleToggle = event => {
+    this.toggled = true
+    this.setState({ toggle: !this.state.toggle })
+    event.stopPropagation()
```

Comment:
I find this a bit confusing. Maybe we can get rid of the toggled flag and implement this as:

```javascript
handleToggle = event => {
  this.setState({ isVisible: !this.state.isVisible })
  event.stopPropagation()
  event.nativeEvent.stopImmediatePropagation()
}

handleDocumentClick = () => {
  this.setState({ isVisible: false })
}
```
?

Reply:
✅ done (b883c48)